src/openrct2/rct1/Tables.cpp

```diff
@@ -686,13 +686,13 @@ namespace RCT1
 
         if (rct1VehicleType == VehicleType::HeartlineTwisterCars)
         {
-            return vehicleSubEntry == HEARTLINE_TWISTER_FORWARDS ? 0 : 1;    
```

Comment:
It appears that my formatter has stripped off some blank lines, should I remove this?

Reply:
It looks like your formatter has removed some trailing whitespace. That is good, actually.